services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -199,6 +199,15 @@ async def stop_dynamic_service(
 
         return RedirectResponse(str(redirection_url))
 
+    if await scheduler.service_awaits_manual_interventions(node_uuid):
+        return JSONResponse(
+            {
+                "code": "waiting_for_intervention",
+                "error": f"Service {node_uuid} waiting for manual intervention",
+            },
+            status_code=status.HTTP_400_BAD_REQUEST,
+        )
```

Comment:
@sanderegg @pcrespov I want this to be a an error response. I am not sure about the returned `status_code`
I need to tell it that this resource is currently waiting for some third party to do something with it and your request will not be accepted.

Reply:
normally for errors you raise  a HTTPException.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
therefore I think I would go with a 409
that would go along the lines of
```python
raise HTTPException(status.HTTP_409_CONFLICT, details="waiting_for_intervention")
```